source/languages/en/riakcs/cookbooks/configuration/configuration-reference.md

```diff
@@ -191,8 +347,70 @@ implement a custom module.</td>
 
 ## Usage Recording
 
-These settings relate to Riak CS's [[access logs|Usage and Billing
-Data]].
+These settings relate to Riak CS's [[access logs|Usage and Billing Data]].
+
+### `riak-cs.conf`
+
+<table class="riak-conf">
+<thead><tr><th>Config</th><th>Description</th><th>Default</th></tr></thead>
+<tbody>
+<tr>
+<td><code>stats.access.archive_period</code></td>
+<td>How large each access archive object is. This setting should be a multiple
+of <code>stats.access.flush_factor</code>. Expressed as a count/time-period
+pair (e.g. 1h (one hour), 60m (60 minutes), and 3600s (3,600 seconds) are all
+equivalent).</td>
+<td><code>1h</code></td>
```

Comment:
How is `1h` acceptable as a count/time-period pair? (What *is* a count/time-period pair?)